src/inc/corinfo.h

```diff
@@ -1568,6 +1568,9 @@ enum CorInfoTokenKind
 
     // token comes from CEE_NEWOBJ
     CORINFO_TOKENKIND_NewObj    = 0x200 | CORINFO_TOKENKIND_Method,
+
+    // token comes from CEE_LDVIRTFTN
+    CORINFO_TOKENKIND_LdvirtualToken = 0x400 | CORINFO_TOKENKIND_Method,
```

Comment:
Nit: the existing values are all named after the IL instruction referring to the token. The consistent naming would be `CORINFO_TOKENKIND_Ldvirtftn`.